lib/views/home/home_view.dart

```diff
@@ -114,6 +117,26 @@ class _HomeViewState extends State<HomeView> {
     return listPayments;
   }
 
+  void getamountMsat() async {
+    final channelsList = await widget.provider.get<AppApi>().listChannels();
+    double totalChannelsAmount = 0;
+    for (var num in channelsList.channels) {
+      totalChannelsAmount += num.amount;
+    }
+
+    /// because all the channnels repeat twice in list channels
+    totalChannelsAmount /= 2;
+
+    /// converting it to sat
+    totalChannelsAmount /= 1000;
+
+    /// converting it to Bitcoin
+    totalChannelsAmount *= 0.00000001;
```

Comment:
We can keep it in satoshi? I have not enough bitcoin in my node to show a decent number :) 

In addition can be set as app setting this if the user want a Bitcoin amount or a sats amount

Reply:
Yes, option for choosing will be better!! But for now let me keep sats

Reply:
>Should I open an issue for this?

Yes better keep track inside a issue